schema/spacewalk/common/tables/suseContentEnvironmentPath.sql

```diff
@@ -0,0 +1,35 @@
+--
+-- Copyright (c) 2018 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+CREATE TABLE suseContentEnvironmentPath(
+    env_id NUMBER
+        CONSTRAINT suse_ct_env_path_eid_fk
+            REFERENCES suseContentEnvironment(id)
+            ON DELETE CASCADE,
```

Comment:
I think we should handle a delete in Java code because it is special.
I think we should not specify a "ON DELETE" action.

Reply:
Ok, I'll remove them. I'd be happier though if the Java code was compatible with these `ON DELETE` actions (both `CASCADE` on `SET NULL` in the next case) to prevent dangling rows in the db.